merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -154,15 +168,17 @@ public List<String> validateModelArguments(final String missionModelId, final Ma
          MissionModelLoadException,
          InvalidArgumentsException
   {
-    return this.loadConfiguredMissionModel(missionModelId)
-               .validateConfiguration(arguments);
+    return this.loadUnconfiguredMissionModel(missionModelId)
+               .validateMissionModelArguments(arguments);
   }
 
+
   @Override
   public List<Parameter> getModelParameters(final String missionModelId)
   throws NoSuchMissionModelException, MissionModelLoadException
   {
-    return loadUnconfiguredMissionModel(missionModelId).getParameters();
+    return this.loadUnconfiguredMissionModel(missionModelId)
+               .getParameters();
```

Comment:
Just to confirm, I think this is only a whitespace change, yes?

Reply:
Yes, my bad - I've gone back and split out whitespace changes into their own commit